samples/msal-node-samples/ExpressTestApp/TestApp/appSettings.json

```diff
@@ -6,8 +6,8 @@
     },
     "settings": {
         "homePageRoute": "/home",
-        "redirectUri": "Enter_the_Redirect_Uri_Here",
-        "postLogoutRedirectUri": "Enter_the_Post_Logout_Redirect_Uri_Here"
+        "redirectUri": "http://localhost:4000/redirect",
```

Comment:
Why are we not leaving this customizable? This mandates the user to have `http://localhost:4000/redirect` as a specific redirectURI to be registered. Is the goal to have a one click working sample?

Reply:
Ah! I see now. We are doing it above but leaving it open here. Never mind.

Reply:
Can we also add some comments, which will mention that this is customizable?

Reply:
Can't put comments in JSON :( 
README mentions its for example though.

Reply:
Makes sense. Resolving.